examples/common/utilities.adb

```diff
@@ -11,6 +11,13 @@ with GL.Types.Colors;
 
 package body Utilities is
 
+   procedure Clear_All (Colour : GL.Types.Colors.Color) is
+   begin
+        GL.Buffers.Clear ((True, True, True, True));
+        GL.Buffers.Set_Color_Clear_Value (Colour);
```

Comment:
Shouldn't these two calls be the other way round? As it is, you first clear the buffers with the previously set clear color, and then you set the new one.